### DISCUSSION TITLE
Не работает команда schema-add

### DISCUSSION THREAD

**Discussion by a1ext0** [totumonline/totum-ru-issues-and-discussions#535] [ОШБК!] [answered]
Здравствуйте! При попытке добавить новую схему в установке multi вылезает следующая ошибка. Пробовал разный формат ввода адреса, использовать доменные имена, вводить логин-пароль или нет. Ошибка сохраняется. В чем может быть проблема?

<img width="531" alt="изображение" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/54662036/daa7169a-e024-4730-b5da-7e46cc2213af">

**Comment by ttm-alexeypolunin** [comment, marked as answer]
На этот вопрос мы наверное не ответим по всяким разным причинам. Посмотрим, возможно во второй половине этого года в V5 (если удасться ее вообще выпустить).

**Reply by ttm-alexeypolunin** [reply]
Проверяли. У нас не повторяется ни в одном сценарии.
